### DISCUSSION TITLE
Is there a way to invoke a shell command during initial set up?

### DISCUSSION THREAD

**Discussion by aarontravass** [vitest-dev/vitest#6247] [Q&A] [answered]
This command runs all migrations files inside the newly spun database (`prisma migrate deploy`) and hence, I was wondering if there was a way to pass this command in the config and have vitest run it during initial set up.

**Comment by sheremet-va** [comment, marked as answer]
You can use `globalSetup` (https://vitest.dev/config/#globalsetup):

```ts
export default defineConfig({
  test: {
    globalSetup: ['./prismaMigrate.ts'],
  },
})
```

```ts
// prismaMigrate.ts
import { x } from 'tinyexec'
export default async () => {
  await x('prisma migrate deploy')
}
```

**Reply by aarontravass** [reply]
Thanks for the reply!
